flo-tests/shading-user/pom.xml

```diff
@@ -10,7 +10,7 @@
 
   <name>Flo Tests - Shading User</name>
   <artifactId>flo-tests-shading-user</artifactId>
-  <version>${flo.version}</version>
+  <version>placeholder-version</version>
```

Comment:
We don't even need this to be the "real" version. It can be any version we want. E.g. `0.0.1-SNAPSHOT` or whatever.

Reply:
Not sure this is causing the build to fail though.